test/e2e/suite/conformance/certificatesigningrequests/acme/acme.go

```diff
@@ -53,6 +53,10 @@ func runACMEIssuerTests(eab *cmacme.ACMEExternalAccountBinding) {
 		featureset.CommonNameFeature,
 		featureset.KeyUsagesFeature,
 		featureset.EmailSANsFeature,
+		featureset.SaveCAToSecret,
+		featureset.IssueCAFeature,
+		featureset.LiteralSubjectFeature,
+		featureset.OtherNamesFeature,
```

Comment:
Should this exactly match the following equivalent list in the certificate conformance tests? 
(why is IPAddressFeature here but not there?)
https://github.com/cert-manager/cert-manager/blob/a7bdbec9e3d674570774248f66f63e9ed7222533/test/e2e/suite/conformance/certificates/acme/acme.go#L50-L63

Reply:
I don't understand why `LiteralSubjectFeature` is even mentioned here. 
Literal subjects are a feature of the Certificate API, not the issuer, right?

In other words, given the explanatory comment above this list:

> // unsupportedHTTP01Features is a list of features that are not supported by the ACME
> // issuer type using HTTP01

....it wouldn't make sense to write 
> The LiteralSubjectFeature is not supported by the ACME issuer using HTTP01


And indeed the list of conformance tests for certificatesigningrequests is much shorter than the tests for certificate,
and doesn't include any  tests that declare they require the literalsubject feature, so that feature shouldn't be included in this list.

Reply:
At some point, we should only have one set of unsupported features (that are used for both Certificates and CertificateSigningRequests).
I'll remove the `LiteralSubjectFeature` feature for now since it is not used by CertificateSigningRequests.